inc/vcf/file_structure.hpp

```diff
@@ -67,160 +67,82 @@ namespace ebi
         NO_VARIATION
     };
 
-    const std::map<std::string, std::pair<std::string, std::string>> info_type_v41_v42 = {
-            { "AA", { "String", "INFO AA metadata Type is not String" } },
-            { "AC", { "Integer", "INFO AC metadata Type is not Integer" } },
-            { "AF", { "Float", "INFO AF metadata Type is not Float" } },
-            { "AN", { "Integer", "INFO AN metadata Type is not Integer" } },
-            { "BQ", { "Float", "INFO BQ metadata Type is not Float" } },
-            { "CIGAR", { "String", "INFO CIGAR metadata Type is not String" } },
-            { "DB", { "Flag", "INFO DB metadata Type is not Flag" } },
-            { "DP", { "Integer", "INFO DP metadata Type is not Integer" } },
-            { "END", { "Integer", "INFO END metadata Type is not Integer" } },
-            { "H2", { "Flag", "INFO H2 metadata Type is not Flag" } },
-            { "H3", { "Flag", "INFO H3 metadata Type is not Flag" } },
-            // TODO : MQ metadata Type is .
-            { "MQ0", { "Integer", "INFO MQ0 metadata Type is not Integer" } },
-            { "NS", { "Integer", "INFO NS metadata Type is not Integer" } },
-            // TODO : SB metadata Type is .
-            { "SOMATIC", { "Flag", "INFO SOMATIC metadata Type is not Flag" } },
-            { "VALIDATED", { "Flag", "INFO VALIDATED metadata Type is not Flag" } },
-            { "1000G", { "Flag", "INFO 1000G metadata Type is not Flag" } }
+    const std::map<std::string, std::pair<std::string, std::string>> info_v41_v42 = {
+            { "AA", { "String", "1" } },
+            { "AC", { "Integer", "A" } },
+            { "AF", { "Float", "A" } },
+            { "AN", { "Integer", "1" } },
+            { "BQ", { "Float", "1" } },
+            { "CIGAR", { "String", "A" } },
+            { "DB", { "Flag", "0" } },
+            { "DP", { "Integer", "1" } },
+            { "END", { "Integer", "1" } },
+            { "H2", { "Flag", "0" } },
+            { "H3", { "Flag", "0" } },
+            { "MQ", { ".", "1" } },
+            { "MQ0", { "Integer", "1" } },
+            { "NS", { "Integer", "1" } },
+            { "SB", { ".", "." } },
```

Comment:
We can't say for sure that SB must be a number, because it is not defined in the specification. So yes, those files should be removed...

On the other hand, check the file referenced in #19, because it could be possible to uncomment it after the last changes you have made. Ignore the comment about checks in the Ragel code because, as you can appreciate, we end up deciding not to do it that way :)